### ISSUE TITLE
bug: Crashes with "panic: set item just set doesn't exist"

### ISSUE THREAD

**Issue by mkubilus** [lacework/terraform-provider-lacework#425]
**Describe the bug**
Updating lacework provider to 1.1.0 crashes with the following stack trace:
```
Stack trace from the terraform-provider-lacework_v1.1.0 plugin:

panic: set item just set doesn't exist

goroutine 29 [running]:
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*MapFieldWriter).setSet(0xc00032a5d0, {0xc00011b3b0, 0x1, 0x1}, {0xca9980, 0xc00013c828}, 0xc0004db040)
        /codefresh/volume/terraform-provider-lacework/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/field_writer_map.go:327 +0x972
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*MapFieldWriter).set(0xc00032a5d0, {0xc00011b3b0, 0x1, 0x1}, {0xca9980, 0xc00013c828})
        /codefresh/volume/terraform-provider-lacework/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/field_writer_map.go:107 +0x14c
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*MapFieldWriter).WriteField(0xc00032a5d0, {0xc00011b3b0, 0x1, 0x1}, {0xca9980, 0xc00013c828})
        /codefresh/volume/terraform-provider-lacework/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/field_writer_map.go:89 +0x3f9
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*ResourceData).Set(0xc0002f8080, {0xe0f0a2, 0xe}, {0xca9980, 0xc00013c828})
        /codefresh/volume/terraform-provider-lacework/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource_data.go:227 +0x210
github.com/lacework/terraform-provider-lacework/lacework.readEcrIam(0xc0002f8080, {0xdde260?, 0xc0001e84e0})
        /codefresh/volume/terraform-provider-lacework/lacework/resource_lacework_integration_ecr.go:590 +0x7c5
github.com/lacework/terraform-provider-lacework/lacework.resourceLaceworkIntegrationEcrRead(0xc0002f8080, {0xdde260, 0xc0001e84e0})
        /codefresh/volume/terraform-provider-lacework/lacework/resource_lacework_integration_ecr.go:240 +0x2f2
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0xf67cc0?, {0xf67cc0?, 0xc0002d7a70?}, 0xd?, {0xdde260?, 0xc0001e84e0?})
        /codefresh/volume/terraform-provider-lacework/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource.go:712 +0x178
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc0004cd6c0, {0xf67cc0, 0xc0002d7a70}, 0xc00045d040, {0xdde260, 0xc0001e84e0})
        /codefresh/volume/terraform-provider-lacework/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource.go:1015 +0x585
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc0004ac528, {0xf67cc0?, 0xc0002d7950?}, 0xc0002ec2c0)
        /codefresh/volume/terraform-provider-lacework/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/grpc_provider.go:613 +0x497
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ReadResource(0xc00047cfa0, {0xf67cc0?, 0xc0002d7170?}, 0xc000300d80)
        /codefresh/volume/terraform-provider-lacework/vendor/github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server/server.go:748 +0x49e
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler({0xdd1120?, 0xc00047cfa0}, {0xf67cc0, 0xc0002d7170}, 0xc00047ae70, 0x0)
        /codefresh/volume/terraform-provider-lacework/vendor/github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:349 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000003c0, {0xf6a940, 0xc000516340}, 0xc0002e2d80, 0xc0004b7590, 0x14ea0f0, 0x0)
        /codefresh/volume/terraform-provider-lacework/vendor/google.golang.org/grpc/server.go:1340 +0xd13
google.golang.org/grpc.(*Server).handleStream(0xc0000003c0, {0xf6a940, 0xc000516340}, 0xc0002e2d80, 0x0)
        /codefresh/volume/terraform-provider-lacework/vendor/google.golang.org/grpc/server.go:1713 +0xa1b
google.golang.org/grpc.(*Server).serveStreams.func1.2()
        /codefresh/volume/terraform-provider-lacework/vendor/google.golang.org/grpc/server.go:965 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /codefresh/volume/terraform-provider-lacework/vendor/google.golang.org/grpc/server.go:963 +0x28a

Error: The terraform-provider-lacework_v1.1.0 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

**To Reproduce**
1. Update lacework provider to 1.1.0
2. Attempt to create a plan that uses this provider

**Expected behavior**
The plugin does not crash

**Screenshots**

**Please complete the following information):**
 - Host OS: Ubuntu 22.04
 - Running within a Docker image of OS: Ubuntu 18.04.6
 - Provider Version: 1.1.0
 - Terraform Version: 1.0.11
 
**Additional context**
This execution is from within a docker container.

**Comment by jon-stewart** [comment]
Hi @mkubilus, could you provide the terraform config/HCL

**Comment by dmurray-lacework** [comment]
Closing this issue, it should be resolved in the 1.2.0 fixed linked by Jon above.
